### ISSUE TITLE
It is not possible to set Kiali's auth strategy via IstioControlPlane API

### ISSUE THREAD

**Issue by cagataygurturk** [istio/istio#19676]
**Bug description**

Setting `values.kiali.auth.strategy` for IstioControlPlane fails

[ ] Configuration Infrastructure
[ ] Docs
[x ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

**Comment by Morriz** [comment]
it is: https://istio.io/docs/reference/config/installation-options/#kiali-options

**Comment by howardjohn** [comment]
I couldn't find the PR, but i have confirmed this is fixed in 1.6:
```
$ ik manifest generate -d manifests --set values.kiali.dashboard.auth.strategy=my-auth-strategy --set profile=demo | grep my-auth-strategy
      strategy: my-auth-strategy
```
